Source/SwiftLintFramework/Rules/RuleConfigurations/TestCaseAccessibilityConfiguration.swift

```diff
@@ -19,6 +21,10 @@ public struct TestCaseAccessibilityConfiguration: RuleConfiguration, Equatable {
         if let allowedPrefixes = configuration["allowed_prefixes"] as? [String] {
             self.allowedPrefixes = Set(allowedPrefixes)
         }
+
+        if let xctestcaseSubclasses = configuration["xctestcase_subclasses"] as? [String] {
+            self.xctestcaseSubclasses = Set(xctestcaseSubclasses)
```

Comment:
You may add `XCTestCase` here directly to the set. That safes us from checking this case explicitly next to the customized test class parents.